ingestion_tools/scripts/importers/collection_metadata.py

```diff
@@ -0,0 +1,8 @@
+from common.finders import DefaultImporterFactory
+from importers.base_importer import BaseFileImporter
+
+
+class CollectionMetadataImporter(BaseFileImporter):
```

Comment:
We should add an entry to https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/99adb45f4c7a42a52da9f7949ee7fc6caedeba03/ingestion_tools/scripts/common/config.py#L243
for `collection_metadata`, with the same value as `rawtilts`.